app/lib/discount.rb

```diff
@@ -0,0 +1,27 @@
+module Discounts
+
+  def discount_total
+    discount = 0
+    discount += discount_total_price
+    discount += discount_muffins
+    discount.round(2)
```

Comment:
This could probably be a seperate object as well, or even a set of objects each of which handles a particular discount